docs/resources/clickhouse.md

```diff
@@ -80,7 +80,7 @@ Optional:
 
 Required:
 
-- `integration_type` (String) Type of the service integration. The only supported value at the moment is `read_replica`
+- `integration_type` (String) Type of the service integration. The only supported values at the moment are `clickhouse_kafka` and `clickhouse_postgresql`.
```

Comment:
Hi @aris-tritas-aiven, this part of the documentation is automatically generated. This part of the text is coming from here: https://github.com/aiven/terraform-provider-aiven/blob/07bd2f737d067e1cf80245408ae518fd18554f72/internal/schemautil/service.go#L164

Reply:
Thank you for the pointer, looking at it